src/main/java/org/example/domain/model/GuildUser.java

```diff
@@ -17,6 +17,8 @@ public class GuildUser {
     private String discordTag;
     @Column(nullable = false)
     private Integer points;
+    @Column(nullable = false)
+    private Boolean insultConsent = false;
```

Comment:
Why not separate fields with endlines ?

Reply:
Why separate fields with newlines?

Reply:
It's more conventional. If someone is used to it, eyes hurt a bit seeing it like this. It's only cosmetic so I'm ok with it from now on.